Fix issue where scrolling was not disabled at page load.

knumat opened the pull request:
After a recent update to Google Calendar, the scrolling was still enabled when the page loaded (see #11 ).  However, the scrolling was properly disabled after the user changed to a different view.

An initial fix was proposed in #12 which was functional, but that doesn't address the root of the problem. Also, the original fix might have a performance impact, since it registers extra "mousewheel" event listeners.

The issue was that the data-viewfamily mutation no longer occurs when the page is loaded.
Now, the data-viewfamily mutation only occurs when the user changes views.
Thus, we need to disable scrolling once when the page is initially loaded.

knumat committed:
Revert "allow the scrolling to be disabled"

This reverts commit a5d5a68ba002df9174480598f874c79686155f4f.

knumat committed:
Fix issue where scrolling was not disabled at page load.

After a recent update to Google Calendar, the scrolling was still working right when the page loaded.
However, the scrolling stopped working after the user changed to a different view.

The issue was that the data-viewfamily mutation no longer occurs when the page is loaded.
Now, the data-viewfamily mutation only occurs when the user changes views.
Thus, we need to disable scrolling once when the page is initially loaded.

mduttondev reviewed (approved):
I like this fix better than mine in #12 . I am admittedly not a web developer, I am a mobile engineer, so HTML and DOM related things are a little out of my wheelhouse. I just went with what I could make functional in my debugging at work.

knumat committed:
Since <body> is available at page load, we can register this handler right away.

knumat committed:
Fix issue where changing months re-enabled scroll wheel.

To do this, we block scroll events on the "main" div instead of the "grid" div.

With the latest version of Google Calendar, the "grid" div is replaced whenever the month is changed.
Specifically, the grid is replaced once the new events have loaded.
Unfortunately, the mutation event occurs before the events have loaded (and before the grid has been replaced).
